src/test/kotlin/de/nordgedanken/auto_hotkey/ide/documentation/AhkDocumentationProviderTest.kt

```diff
@@ -0,0 +1,155 @@
+package de.nordgedanken.auto_hotkey.ide.documentation
+
+import com.intellij.ide.BrowserUtil
+import com.intellij.openapi.projectRoots.impl.ProjectJdkImpl
+import com.intellij.psi.PsiElement
+import de.nordgedanken.auto_hotkey.AhkBasePlatformTestCase
+import de.nordgedanken.auto_hotkey.ProjectDescriptor
+import de.nordgedanken.auto_hotkey.WithOneAhkSdkAsProjDefault
+import de.nordgedanken.auto_hotkey.lang.core.AhkFileType
+import de.nordgedanken.auto_hotkey.project.settings.defaultAhkSdk
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import io.kotest.matchers.string.shouldContain
+import io.mockk.Runs
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockkStatic
+import io.mockk.verify
+import org.junit.jupiter.api.Test
+import java.io.File
+
+class AhkDocumentationProviderTest : AhkBasePlatformTestCase() {
+
+    override fun getTestDataPath(): String = File("src/test/resources/").absolutePath
```

Comment:
@amynbe you marked the conversation on this line as resolved, but it does not look like any code change was actually pushed. Please verify that the change was pushed?

(I summarized the conversation below as a reference:)

---
The path to the test resources directory is used in multiple other test classes, so a constant has already been created for it in the root test class. Please use the constant below so that we are not duplicating this String in multiple locations:

```kotlin
override fun getTestDataPath() = AhkTestCase.testResourcesPath
```

Reply:
My bad. Fixed now.